src/server/status/routes/api/register_stats.js

```diff
@@ -0,0 +1,33 @@
+import { wrapAuthConfig } from '../../wrap_auth_config';
+
+export function registerStatsApi(kbnServer, server, config, collector) {
+  const wrapAuth = wrapAuthConfig(config.get('status.allowAnonymous'));
+
+  server.route(wrapAuth({
+    method: 'GET',
+    path: '/api/stats',
+    config: {
+      tags: ['api']
+    },
+    async handler(request, reply) {
+
+      let clusterUuid;
+      try {
+        const { callWithRequest } = request.server.plugins.elasticsearch.getCluster('data');
+        const { cluster_uuid: uuid } = await callWithRequest(request, 'info', { filterPath: 'cluster_uuid' });
```

Comment:
it does.  thoughts on a different endpoint if we need the cluster uuid?  it looks like these are all independent of the cluster except for uuid, and it adds a network request to in memory data

Reply:
Yeah, hm. I like the idea of not making that request to ES if the user doesn't care about cluster_uuid in the first place.

How keep the same endpoint, and keep it as a GET, but add an optional query param? `/api/stats?with_cluster_uuid` or something?

Reply:
@ruflin does it make a difference on the Beats side whether fetching the cluster_uuid is opt-in or opt-out? I could make a separate endpoint completely, but it seems like over-engineering - it'd have to be named and documented reasonably.

Reply:
++ for `extended`. Thanks!

Reply:
@tsullivan We can make the field optional and then it's fetched when it's there and otherwise ignored. I like it to have in the same document as it means we don't have to fire an additional http request.

If we have an `extended` mode an additional endpoint could make more sense, but param for 1 endpoint also works.

@tsullivan For the ES cluster_id: Can KB be connected to more then one cluster in the future? And could we perhaps cache the ES cluster uuid locally and only update it if KB for example reconnets to ES and check if it's still the same (if there is such a thing like reconnect).